packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -2149,6 +2143,17 @@ describe('network stubbing', { retries: 2 }, function () {
       })
     })
 
+    // @see https://github.com/cypress-io/cypress/issues/9062
+    it('can spy on a request using forceNetworkError', function () {
+      cy.intercept('/foo', { forceNetworkError: true })
+      .as('err')
+      .then(() => {
+        $.get('/foo')
+      })
+      .wait('@err').should('have.property', 'error')
```

Comment:
what is the value of that `error` property?

Reply:
the `Error` object

Reply:
Perfecto